gobblin-api/src/main/java/gobblin/configuration/ConfigurationKeys.java

```diff
@@ -486,6 +486,14 @@
   public static final String ENCRYPT_KEY_LOC = "encrypt.key.loc";
 
   /**
+   * Proxy Filesystem operation properties.
+   */
+  public static final String FS_AS_PROXY_USER = "fs.as.proxy.user";
```

Comment:
I am using the same key for deciding if we need proxy in `publisher`. What do you think if I change it to `should.fs.proxy.as.user`?